src/master/filesystem_dump.cc

```diff
@@ -90,7 +91,10 @@ void fs_dumpnode(FSNode *f) {
 		printf("|d:%5" PRIu32 ",%5" PRIu32 "\n", static_cast<FSNodeDevice*>(f)->rdev >> 16,
 		       static_cast<FSNodeDevice*>(f)->rdev & 0xFFFF);
 	} else if (f->type == FSNodeType::kSymlink) {
-		printf("|p:%s\n", fsnodes_escape_name((std::string)static_cast<FSNodeSymlink*>(f)->path).c_str());
+		printf("|p:%s\n",
+		       gFSOperations->nodeOperations()
+		           ->fsnodes_escape_name((std::string) static_cast<FSNodeSymlink *>(f)->path)
```

Comment:
Consider this more idiomatic approach.
```suggestion
gFSOperations->nodeOperations()
    ->fsnodes_escape_name(std::string{static_cast<FSNodeSymlink*>(f)->path});
```

Reply:
I will do it in the refactor.
Anyway, please notice that the proposed change is creating a copy of the string.